### PR TITLE
Feature/issue 152 polish my pan

### PR DESCRIPTION
**Pull request by rasmus-storjohann-PG**

### CODE REVIEW

**Comment by rasmus-storjohann-PG** [comment]
@bensinjin for review, this takes care of the My Plan page. There is one outstanding bug but I'd like to get this in and track that bug separately, I've got too many big PRs in play.

**Comment by rasmus-storjohann-PG** [comment]
This PR requires a server side change, see https://github.com/pg-irc/pathways-backend/pull/211/files

File: src/components/tasks/task_list_item.tsx [hunk 1]
```diff
@@ -45,8 +45,8 @@ export const TaskListItemComponent: React.StatelessComponent<AllTaskListItemProp
 const renderTaskInteractions = (props: AllTaskListItemProps): JSX.Element => {
     const addToSavedList = (): AddToSavedListAction => props.addToSavedList(props.id);
     return (
-        <Col size={10}>
-            {props.addToSavedList ? renderAddButton(addToSavedList) : renderDragButton()}
+        <Col size={15}>
+            {props.addToSavedList ? renderAddButton(addToSavedList) : renderUnchecedkBox()}
```
Comment (bensinjin) [inline src/components/tasks/task_list_item.tsx]: Fix typo, should be `renderUncheckedBox()`

File: src/components/tasks/task_list_item.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ const renderAddButton = (onPress: () => void): JSX.Element => (
     </Button>
 );
 
-const renderDragButton = (): JSX.Element => (
+const renderUnchecedkBox = (): JSX.Element => (
```
Comment (bensinjin) [inline src/components/tasks/task_list_item.tsx]: So I'm noticing I still see the "+" when the task is in my "TASKS I PLAN TO DO" section. Is this the bug you're referring to? It stems from the fact that the "+" is rendered whenever `props.addToSavedList` is defined.

File: src/components/tasks/task_list_item.tsx (hunk 1 above)
Comment (rasmus-storjohann-PG) [reply src/components/tasks/task_list_item.tsx]: Fixed

File: src/components/tasks/task_list_item.tsx (hunk 2 above)
Comment (rasmus-storjohann-PG) [reply src/components/tasks/task_list_item.tsx]: Yeah, looking at it briefly it seems to me that the fix won't be completely simple, so I'd rather get this PR merged and fix it later, probably under the same story.

Comment (rasmus-storjohann-PG) [reply src/components/tasks/task_list_item.tsx]: There are also some strings that aren't translated, we can do a full sweep of the app for that later.

**Comment by rasmus-storjohann-PG** [comment]
Rebased. Remember to merge the server side piece at the same time

**Pull request merged**